openapi_codec/__init__.py

```diff
@@ -24,12 +23,15 @@ def load(self, bytes, base_url=None):
         except ValueError as exc:
             raise ParseError('Malformed JSON. %s' % exc)
 
+        base_url = options.get('base_url')
         doc = _parse_document(data, base_url)
         if not isinstance(doc, Document):
             raise ParseError('Top level node must be a document.')
 
         return doc
 
-    def dump(self, document, **kwargs):
+    def encode(self, document, **options):
+        if not isinstance(document, Document):
+            raise ValueError('Expected a `coreapi.Document` instance')
```

Comment:
This sounds like a [`TypeError`](https://docs.python.org/2/library/exceptions.html#exceptions.TypeError).